src/App.js

```diff
@@ -1,18 +1,21 @@
 import MoviesBrowser from "./features/MoviesBrowser";
 import Navigation from "./common/Navigation";
-import { BrowserRouter } from "react-router-dom";
+import React from "react";
+import { BrowserRouter, Route, Switch } from "react-router-dom";
+import { PeopleList } from "./features/PeopleList";
```

Comment:
I believe PeopleList should be exported as default (in it's component) and them imported without the curly braces. It would be much better practice, since we already use that convention in the project 😎